src/js/components/DescriptionList.js

```diff
@@ -1,6 +1,12 @@
+import PureRenderMixin from 'react-addons-pure-render-mixin';
 import React from 'react';
 
 class DescriptionList extends React.Component {
+  constructor() {
+    super(...arguments);
+    this.shouldComponentUpdate = PureRenderMixin.shouldComponentUpdate;
```

Comment:
I don't think so. `DescriptionList.shouldComponentUpdate`as defined there is properly scoped.